inc/theme.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Theme setup.
+ *
+ * @package Create_WordPress_Theme
+ */
+
+namespace Create_WordPress_Theme;
+
+add_action( 'after_setup_theme', __NAMESPACE__ . '\action__after_theme_setup' );
+
+/**
+ * Setup theme defaults and registers support for various WordPress features.
+ */
+function action__after_theme_setup(): void {
+	/**
+	 * Add menu support.
+	 */
+	add_theme_support( 'menus' );
+
+	/**
+	 * Enable theme customizer.
+	 */
+	add_action( 'customize_register', '__return_true' );
```

Comment:
This does seem to be an appropriate solution per https://developer.wordpress.org/reference/hooks/customize_register/#comment-6395

Reply:
That comment on wordpress.org is accurate, as far as it goes, but I think the bot is more correct. This isn't really the correct way to use `customize_register`, not least because actions don't return values. 

Taking a step back, why does the Customizer need to be enabled in this theme?

Reply:
@dlh01 I think the reasoning behind having the customizer in place is that it allows for things like setting a Site Icon, etc. @kevinfodness would you like to weigh in here? If we want to keep it activated, how would you recommend we do that?